tensorflow/lite/kernels/internal/reference/reduce.h

```diff
@@ -23,6 +23,26 @@ limitations under the License.
 #include "tensorflow/lite/kernels/internal/quantization_util.h"
 #include "tensorflow/lite/kernels/internal/types.h"
 
+namespace {
+
+bool IsFirstReduction(const int* index, const int num_axis, const int* axis) {
```

Comment:
Nit: could you please add a short comment for this function?

Reply:
Done